evap/staff/templates/staff_course_form.html

```diff
@@ -85,7 +85,7 @@
 
```

Comment:
why would you want to have the message? that doesn't make sense to me.
i would never use the save and approve button in this situation because i would always want to check the preview first, but if you insist i can think about adding it.